api/jest.utils.ts

```diff
@@ -0,0 +1,30 @@
+import request from 'supertest';
+
+import { build } from './src/app';
+
+declare global {
+  // eslint-disable-next-line no-var
+  var fastifyTestInstance: Awaited<ReturnType<typeof build>> | undefined;
+}
+
+export function superGet(endpoint: string): request.Test {
+  return request(fastifyTestInstance?.server)
+    .get(endpoint)
+    .set('Origin', 'https://www.freecodecamp.org');
+}
```

Comment:
When/why/how/where will we be running Jest in production?

Reply:
Never, but we can mock the FCC_NODE_ENV value.

It just bugged me that I couldn't test the behaviour I was interested in (i.e. production), so now we can.

I expect to refine/abandon this as we write more tests, though.